app/app.go

```diff
@@ -213,7 +221,7 @@ func New(
 
 	base := baseapp.NewBaseApp(Name, logger, db, encodingConfig.TxConfig.TxDecoder(), baseAppOptions...)
 	base.SetCommitMultiStoreTracer(traceStore)
-	base.SetAppVersion(version.Version)
```

Comment:
id recommend removing this until state sync is fixed.

Reply:
Thanks @marbar3778!!!